packages/classic-webview/src/pages/UnlockHardcorePage.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { HardcoreLogo } from '@hotandcold/webview-common/components/logo';
+import { GoldIcon } from '@hotandcold/webview-common/components/icon';
+
+const primaryBorderColor = '#FFBF0B';
+const secondaryBorderColor = '#8BA2AD';
+const secondaryPillBgColor = '#2A3236';
+
+// Inline PurchaseButton component
+interface PurchaseButtonProps {
+  mainText: string;
+  style: 'primary' | 'secondary';
+  badgeIcon: React.ReactNode;
+  badgeText: string;
+  onClick?: () => void;
+}
+
+const PurchaseButton: React.FC<PurchaseButtonProps> = (props) => {
+  const { mainText, badgeIcon, badgeText, onClick, style } = props;
+  const color = style === 'primary' ? primaryBorderColor : secondaryBorderColor;
+  const pillBg = style === 'secondary' ? secondaryPillBgColor : color;
+  const pillTextColor = style === 'secondary' ? 'white' : 'black';
+
+  return (
+    <button
+      onClick={onClick}
+      className="flex w-full flex-auto flex-row items-center justify-between gap-4 whitespace-nowrap rounded-[64px] border-2 p-3 sm:w-auto"
+      style={{ borderColor: color }}
+    >
+      <span
+        className="text-left font-sans text-base font-semibold leading-5 tracking-tight"
+        style={{ color }}
```

Comment:
Let's avoid using styles. You should be able to accomplish the same thing with a tailwind class name.